infra/prod/_modules/functions/function_relying_party.tf

```diff
@@ -0,0 +1,85 @@
+locals {
+  relying_party_func_settings = {
+    for s in var.relying_party_func.app_settings :
+    s.name => s.key_vault_secret_name != null ? "@Microsoft.KeyVault(VaultName=${var.product}-kv;SecretName=${s.key_vault_secret_name})" : s.value
+  }
+}
+
+module "relying_party_func" {
+  source = "github.com/pagopa/terraform-azurerm-v3.git//function_app?ref=v7.72.2"
+
+  name                = format("%s-relying-party-func", var.project)
+  location            = var.location
+  resource_group_name = var.resource_group_name
+
+  health_check_path = "/health"
+
+  app_settings = local.relying_party_func_settings
+
+  node_version    = "18"
+  runtime_version = "~4"
+  always_on       = true
+
+  app_service_plan_info = {
+    kind                         = "Linux"
+    sku_size                     = "P1v3"
```

Comment:
@Krusty93 `sku` check please! 😄 
is that an appropriate `sku` for now? (pre-release?)

Reply:
as a side note for @Krusty93 these values should go as default inside the function module, we could limit them to max 3 tiers (ie. small, medium, large)